api/service.py

```
@@ -741,6 +779,17 @@ async def get_status(api_key: APIKey = Depends(get_api_key)):
     )
 
 
+@app.get("/api/v1/resources/health")
+async def get_resource_health(api_key: APIKey = Depends(get_api_key)):
```

Ruff (0.14.10)

Unused function argument: `api_key`

(ARG001)